composer.json

```diff
@@ -10,17 +10,16 @@
     }
   ],
   "require": {
-    "php": "~7.2|~7.3|~7.4|~8.1",
-    "psr/log": "^1.0",
-    "beberlei/assert": "^2.4|^3.0",
-    "regex-guard/regex-guard": "^1.1"
+    "php": "8.0.* || 8.1.*",
+    "psr/log": "^3.0",
```

Comment:
Is there a chance to make it compatible with all psr/log versions? Just thinking about compatibility with other implementations …

GitHub Actions should provide ne necessary testing mechanisms (locked, lowest, update)